lib/rules/landmark-unique-matches.js

```diff
@@ -4,17 +4,18 @@ import { getRole } from '../commons/aria';
 import { getAriaRolesByType } from '../commons/standards';
 import { accessibleTextVirtual } from '../commons/text';
 
+// Sectioning content -
+// https://html.spec.whatwg.org/multipage/dom.html#sectioning-content
+const sectioningContentElements = ['article', 'aside', 'nav', 'section'];
+
 /*
- * Since this is a best-practice rule, we are filtering elements as dictated by ARIA 1.1 Practices regardless of treatment by browser/AT combinations.
- *
- * Info: https://www.w3.org/TR/wai-aria-practices-1.1/#aria_landmark
```

Comment:
This comment didn't make that much sense to me and the link was outdated, which is why I changed it.